examples/ec2/versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.1"
+  required_version = ">= 1.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 2.48"
+      version = ">= 4.6"
```

Comment:
why v4.6 - [this](https://github.com/hashicorp/terraform-provider-aws/pull/23600) is the latest change for ECS service which was released in v4.6. The service sub-module is coming up next

Reply:
I agree. It is totally ok to rely on a newer version in such a major release.